course3/homework-vector/vector.cc

```diff
@@ -1,9 +1,53 @@
 #include <iostream>
 #include <cassert>
+#include <cmath>
 
 class Vector
 {
-    // TODO
+    private:
+        int private_x;
+        int private_y;
+
+    public:
+        // required constructors
+        Vector(int x, int y): private_x(x), private_y(y) {}
+        Vector() = default;
```

Comment:
The value of default-initialized int is unspecified. There is something called "zero initialization", but this is not what's happening here.
http://en.cppreference.com/w/cpp/language/default_initialization